src/backend/columnar/sql/downgrades/columnar--11.1-3--11.1-2.sql

```diff
@@ -0,0 +1,211 @@
+-- columnar--11.1-3--11.1-2.sql
+
+DROP FUNCTION pg_catalog.alter_table_set_access_method;
```

Comment:
I tried `ALTER FUNCTION columnar.alter_table_set_access_method RENAME TO pg_catalog.alter_table_set_access_method` and pg doesn't allow me to rename a function with the `.` prefix